### DISCUSSION TITLE
Allow users to set certain skin elements to be consistent across all skins

### DISCUSSION THREAD

**Discussion by Hiviexd** [ppy/osu#19942] [Ideas] [answered]
not sure if the title's wording is the best, but basically this means that I could choose a certain skin element (like a cursor or a UR bar) which will stay the same across all skins.

this would circumvent the issue of having to consistently edit many skins manually, or making multiple copies of a skin only for minor changes.

**Comment by smoogipoo** [comment, marked as answer]
Sounds like basically the same as, or a roundabout way of achieving the same result as https://github.com/ppy/osu/discussions/19895. See the response in that thread.
